provisioning/cli-dist/release_role.tf

```diff
@@ -0,0 +1,65 @@
+# IAM role which allows production release workflow push artifacts to cli-dist S3 bucket
+
+data "aws_iam_policy_document" "cli_dist_release_assume_policy_doc" {
+  version = "2012-10-17"
+  statement {
+    sid = "AllowAssumeRoleFromGithub"
+    actions = [
+      "sts:AssumeRoleWithWebIdentity"
+    ]
+    effect = "Allow"
+    principals {
+      identifiers = [
+        var.github_oidc_provider_arn
+      ]
+      type = "Federated"
+    }
+    condition {
+      test     = "StringLike"
+      variable = "token.actions.githubusercontent.com:sub"
+      values = [
+        "repo:keboola/keboola-as-code:ref:refs/tags/*",
```

Comment:
Vím že se plánuje přejmenování repa, nevím ale jestli dává smysl vytáhnout do variable...

Reply:
Asi to nemusime teraz riesit, v tomto Q4 sa to premenovavat myslim nebude